docs/01_cva6_user/Interfaces.rst

```diff
@@ -0,0 +1,49 @@
+..
+   Copyright (c) 2023 OpenHW Group
+   Copyright (c) 2023 Thales
+
+   SPDX-License-Identifier: Apache-2.0 WITH SHL-2.1
+
+.. Level 1
+   =======
+
+   Level 2
+   -------
+
+   Level 3
+   ~~~~~~~
+
+   Level 4
+   ^^^^^^^
+
+.. _cva6_interfaces:
+
+Interfaces
+==========
+
+AXI Interface
```

Comment:
AXI interface chapter is big, can you create a dedicated file for it ?  When done, I will push the file

Reply:
In this AXI section, I plan to point on the specificities of the interface (USER, atomic support), so I think it will be quite compact.
The list of signals will be on the prior section.
For the AXI specification (chronograms, transactions...), we'll point to ARM's references.
If a chapter gets too big, we can always decide to cut it later.